src/main/scala/it/pagopa/interop/partyprocess/api/impl/package.scala

```diff
@@ -64,7 +64,7 @@ package object impl extends DefaultJsonProtocol {
   implicit val institutionFormat: RootJsonFormat[Institution]           = jsonFormat20(Institution)
   implicit val institutionSeedFormat: RootJsonFormat[InstitutionSeed]   = jsonFormat15(InstitutionSeed)
   implicit val productInfoDataFormat: RootJsonFormat[ProductInfo]       = jsonFormat3(ProductInfo)
-  implicit val onboardingDataFormat: RootJsonFormat[OnboardingData]     = jsonFormat17(OnboardingData)
+  implicit val onboardingDataFormat: RootJsonFormat[OnboardingData]     = jsonFormat22(OnboardingData)
```

Comment:
Take in consideration that you just hit the limit, you can't add other named fields if not grouping them in subobjects

Reply:
I agree with @TonioGela . 
Use grouping objects where possible.